var/spack/repos/builtin/packages/libmbd/package.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Libmbd(CMakePackage):
```

Comment:
You can have a look at `gdal` to check how to build python wrappers on a `CMakePackage`. The base class `PythonExtension` should contain already a few useful methods.